packages/base/core/src/less/_variables.less

```diff
@@ -219,6 +219,15 @@
 @oui-choice-thumbnail-border-color_disabled: @oui-color-charlotte;
 @oui-choice-thumbnail-border-color_error: @oui-color-error-light;
 
+@oui-choice-thumbnail-footer-background: @oui-color-charlotte;
```

Comment:
If it's only available for the radio component, this should be moved in the radio variables ;)
`oui-choice` is the common style used by `oui-radio`, `oui-checkbox` and `oui-select-picker`.